pig/src/main/java/org/jboss/pnc/bacon/pig/Pig.java

```diff
@@ -100,14 +100,14 @@ private Pig() {
         @Option(
                 names = { "-t", "--tempBuild" },
                 defaultValue = "false",
-                description = "If specified, PNC will perform temporary builds. Akin to dry-run but with temporary alignment preference.")
+                description = "If specified, PNC will perform temporary builds.")
         boolean tempBuild;
 
         @Option(
-                names = "--dry-run",
-                defaultValue = "false",
-                description = "If specified, PNC will perform a dry run. Akin to tempBuild but with persistent alignment preference.")
-        boolean dryRun;
+                names = { "--tempBuildAlignment" },
+                defaultValue = "TEMPORARY",
+                description = "Alignment preference for temporary build. Options: (default)TEMPORARY, PERSISTENT")
+        String tempAlign;
```

Comment:
I think this should be an enum I.e. https://picocli.info/#_enum_types
This would also simplify some of the following code and provide better error checking

Reply:
changed :+1: